miner/blockoracle.go

```diff
@@ -16,20 +16,21 @@ type activationDB interface {
 	GetNodeAtxIDForEpoch(nodeID types.NodeID, targetEpoch types.EpochID) (types.ATXID, error)
 	GetAtxHeader(id types.ATXID) (*types.ActivationTxHeader, error)
 	GetIdentity(edID string) (types.NodeID, error)
+	GetEpochAtxs(epochID types.EpochID) (atxs []types.ATXID)
 }
 
-type signer interface {
+type vrfSigner interface {
 	Sign(msg []byte) ([]byte, error)
 }
 
-// MinerBlockOracle is the oracle that provides block eligibility proofs for the miner.
-type MinerBlockOracle struct {
+// BlockOracle is the oracle that provides block eligibility proofs for the miner.
+type BlockOracle struct {
```

Comment:
I would even name this `Oracle`. since it is now in the `miner` package. it'll be `miner.Oracle` which is nice.